### ISSUE TITLE
Add COLOR specification to ICS files

### ISSUE THREAD

**Issue by keunes** [AntennaPod/antennapod.github.io#202]
**Short description**: Specify colour for ICS files (calendar & event level).

**Location**: 

**Why have this**: To support the AntennaPod brand also in users' other apps.

**More info**: 
* https://icalendar.org/New-Properties-for-iCalendar-RFC-7986/5-9-color-property.html
* https://www.cssportal.com/css3-color-names/

**Comment by ByteHamster** [comment]
Per event? Not sure about that. I don't want the AntennaPod calendar to mess with my entire calendar because it has events in the colors of other calendars. Having only one single color for the entire calendar is more predictable.

**Comment by keunes** [comment]
Yes, sorry, I meant on the individual event ICS files (on calendar level) when we get there.
